docs/en/advanced_guides/data_flow.md

```diff
@@ -1 +1,96 @@
 # Data Flow
+
+- [Data Flow](#data-flow)
+  - [Data flow between dataloader and model](#data-flow-between-dataloader-and-model)
+    - [Data from dataset](#data-from-dataset)
+    - [Data from dataloader](#data-from-dataloader)
+    - [Data from data preprocessor](#data-from-data-preprocessor)
+
+Data flow defines how data should be passed between two isolated modules, e.g. dataloader and model, as shown below.
+
+<div align="left">
+<img src="https://user-images.githubusercontent.com/30762564/185855134-89f5be9e-39ca-4da4-bd87-7cf26e80ab2f.png" width="40%"/>
```

Comment:
'width=40%' is not very clear
![image](https://user-images.githubusercontent.com/36138628/187020154-772b1478-985a-4539-9e75-b690cb6a72ca.png)